app/assets/javascripts/components/sidebar/posts_main_component.js.coffee

```diff
@@ -0,0 +1,44 @@
+# @state.posts is an array of json objects describing posts
```

Comment:
this comment is unnecessary and was actually false at the time it was written (because `@state.posts` didn't exist and there was `@state.posts_json`, so it can safely be removed because the code makes it clear